README.md

```
@@ -1,3 +1,18 @@
+Bootstrap
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)